securityconfig/roles.yml

```diff
@@ -129,6 +129,38 @@ asynchronous_search_read_access:
   cluster_permissions:
     - 'cluster:admin/opendistro/asynchronous_search/get'
 
+# Allows user to use all index_management actions - ism policies, rollups, transforms
+index_management_full_access:
+  reserved: true
+  cluster_permissions:
+    - "cluster:admin/opendistro/ism/policy/write"
+    - "cluster:admin/opendistro/ism/policy/search"
+    - "cluster:admin/opendistro/ism/policy/get"
+    - "cluster:admin/opendistro/ism/policy/delete"
+    - "cluster:admin/opendistro/ism/managedindex/add"
+    - "cluster:admin/opendistro/ism/managedindex/remove"
+    - "cluster:admin/opendistro/ism/managedindex/change"
+    - "cluster:admin/opendistro/ism/managedindex/retry"
+    - "cluster:admin/opendistro/ism/managedindex/explain"
+    - "cluster:admin/opendistro/rollup/index"
```

Comment:
shouldn't these rollup and transform be a part of ISM plugin and should have same path as of policy and mangedindex ?

ideally it should be like .. ```ism/rollup``` and ```ism/transform``` same as ```ism/policy``` and ```ism/managedindex```

Reply:
Created an issue in IndexManagement https://github.com/opensearch-project/index-management/issues/138